packages/hash/shared/src/blocks.ts

```diff
@@ -231,8 +231,11 @@ export const fetchBlock = async (
   return await promise;
 };
 
+export const paragraphBlock =
```

Comment:
Suggest `paragraphBlockId` or `paragraphBlockComponentId`

Reply:
I used `paragraphBlock` to keep the naming consistent with the other names in the same file.
Check the following variables, maybe we should update them as well? @CiaranMn 
https://github.com/hashintel/hash/blob/0af87c3258d57211df64bcfcf9f06ead1ab0a949/packages/hash/shared/src/blocks.ts#L237-L241
https://github.com/hashintel/hash/blob/0af87c3258d57211df64bcfcf9f06ead1ab0a949/packages/hash/shared/src/blocks.ts#L251-L260

Reply:
Yes – I think `defaultBlocksComponentIds`, `textBlockComponentIds`, `paragraphComponentId` would be clearer

Reply:
Updated variable names here https://github.com/hashintel/hash/pull/953/commits/43e136eef1de516c16cd8c0d818b0d0434dd024d